src/avatar.cpp

```diff
@@ -1461,6 +1461,19 @@ bool avatar::invoke_item( item *used, const std::string &method )
     return Character::invoke_item( used, method );
 }
 
+void avatar::update_cardio_acc()
```

Comment:
Is this function ever called? I don't see any invocations; this looks like dead code.

Reply:
Afaict that's an error if so, I wonder if it got missed during one of the conflict merges

Reply:
I see now the comment from gkarfakis19 on the previous unfinished PR (repeated several times):

> I do NOT plan on adding the exercise (cardio_acc) aspect of Cardio in this PR. It will come later.

It looks like it was implemented as requested, just not hooked up to anything. As noted in my other comment on this function, I think it would make sense to invoke it from `Character::update_body` just before the `advance_daily_calories()` call; that way it ought to include the previous 24 hours of activity from the diary.